xonsh/__init__.py

```diff
@@ -26,8 +26,6 @@
         _sys.modules['xonsh.platform'] = __amalgam__
         pretty = __amalgam__
         _sys.modules['xonsh.pretty'] = __amalgam__
-        jobs = __amalgam__
```

Comment:
@scopatz is the amalgam serialisation orders not reliable, or am I doing something wrong?